_episodes/40.md

```diff
@@ -0,0 +1,481 @@
+<img align="center" width="50%" height="100%" src="/assets/episode/15/trino-iceberg.png"/><br/>
+Looks like Commander Bun Bun is safe on this Iceberg<br/>
+<a href="https://joshdata.me/iceberger.html">https://joshdata.me/iceberger.html</a>
+</p>
+
+## Hosts
+
+ * Brian Olsen, Developer Advocate at [Starburst](https://starburst.io)
+ ([@bitsondatadev](https://twitter.com/bitsondatadev))
+ * [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate at 
+ [Starburst](https://starburst.io)
+
+## Guests
+
+ * Ryan Blue, creator of Iceberg and CEO at 
+ [Tabular](https://tabular.io) ([@rdblue](https://github.com/rdblue))
+ * Sam Redai, Developer Advocate at [Tabular](https://tabular.io)
+ ([@samuelredai](https://twitter.com/samuelredai))
+ * [Tom Nats](https://www.linkedin.com/in/tomnats), Director of Customer Solutions at [Starburst](https://starburst.io)
+
+## Register for Trino Summit 2022!
+
+Trino Summit 2022 is coming around the corner! The free event on November 10th 
+and will take place in-person at the Commonwealth Club in San Francisco, CA or
+can also be attended remotely!  If you want to present, the 
+[call for speakers](https://sessionize.com/trino-summit-2022/) is open until
+September 15th. 
+
+[You can register for the conference](https://www.starburst.io/info/trinosummit/)
+at any time. We must limit in-person registrations to 250 
+attendees, so register soon if you plan on attending in person!
+
+## Releases 394 to 395
+
+Official highlights from Martin Traverso:
+
+[Trino 394]({{site.url}}/docs/current/release/release-394.html)
+
+* JSON output format for EXPLAIN.
+* Improved performance for LIKE expressions.
+* `query` table function in BigQuery connector.
+* INSERT support in BigQuery connector.
+* TLS support in Pinot connector.
+
+[Trino 395]({{site.url}}/docs/current/release/release-395.html)
+
+* Faster INSERT queries.
+* Better performance for large clusters.
+* Improved memory efficiency for aggregations and fault tolerant execution.
+* Faster aggregations over DECIMAL columns.
+* Support for dynamic function resolution.
+
+Additional highlights worth a mention according to Cole:
+
+* The improved performance of inserts on Delta Lake, Hive, and Iceberg is a huge
+one. We're not entirely sure how much it'll matter in production use cases, but
+some of the benchmarks suggested it could be massive - one test showed a 75%
+reduction in query duration.
+* Dynamic function resolution in the SPI is going to unlock some very neat
+possibilities down the line. 
+
+More detailed information is available in the release notes for
+[Trino 394]({{site.url}}/docs/current/release/release-394.html),
+and
+[Trino 395]({{site.url}}/docs/current/release/release-395.html).
+
+## Concept of the week: Latest features in Apache Iceberg and the Iceberg connector
+
+It has been over a year since we had Ryan on the Trino Community Broadcast as
+guest to discuss what Apache Iceberg is, and how it can be used in Trino. Since
```

Comment:
```suggestion
guest to discuss what Apache Iceberg is and how it can be used in Trino. Since
```
"how it can be used in Trino" isn't an independent clause, so no comma is necessary before the "and"